packages/cdktf/lib/terraform-variable.ts

```diff
@@ -86,6 +91,11 @@ export interface TerraformVariableConfig {
    * The nullable argument in a variable block controls whether the module caller may assign the value null to the variable.
    */
   readonly nullable?: boolean;
+
+  /**
+   * Specify arbitrary custom validation rules for a particular variable using a validation block nested within the corresponding variable block
+   */
+  readonly validation?: TerraformVariableValidationConfig;
```

Comment:
This should probably take an (optional) array of `TerraformVariableValidationConfig`, as it is valid to pass multiple validation blocks (see bottom of this [docs section](https://www.terraform.io/language/values/variables#custom-validation-rules)).

Reply:
My bad, done and added logic!